src/Microsoft.AspNet.IISPlatformHandler/IISPlatformHandlerMiddleware.cs

```diff
@@ -83,13 +84,22 @@ private static void UpdateRemoteIp(HttpContext httpContext)
             if (xForwardedForHeaderValue != null && xForwardedForHeaderValue.Length > 0)
```

Comment:
Is it time to make this easier to read with `var connection = httpContext.Connection;`?